src/customHooks/useProducts.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import { ProductType } from '../types';
+
+export default (initialValue: ProductType[]) => {
+  const [ products, setProducts ] = useState<ProductType[]>(initialValue);
+
+  // REMOVE DUPLICATE, HIGHER LEVEL CATEGORY (NON-PRODUCE/PRODUCE)
+  const parsedProducts = (newProducts: ProductType[]) => { 
+    return newProducts.map((product) => {
+      product.categories = product.categories.filter(category => !['PRO', 'DRY'].includes(category.commonId));
+      return product;
```

Comment:
I'm personally not a big fan of mutating function parameters like this. I personally like to see things more like this:

```
{...product, categories: product.categories.filter....}
```

This avoids a chance of confusion down the road of execution when you're wondering *where* something was changed and how.